spec/acceptance/enum_traits_spec.rb

```diff
@@ -1,9 +1,17 @@
 describe "enum traits" do
+  def define_model_with_enum(class_name, field, values)
+    define_model(class_name, status: :integer) do
+      if ActiveRecord::VERSION::STRING >= "7.2"
```

Comment:
```suggestion
      if ActiveRecord::VERSION::STRING >= "7.0"
```

While the old syntax was removed on Rails 7.2, the new one was introduced on 7.0. By referencing the version when it was added we can more easily remove the old syntax when we drop support for Rails < 7.0.

(Rails 6.1 will stop receiving security updates in a month or so, by the way)

Reply:
Oh, it was just deprecated, not removed. I thought it was deprecated when the new syntax landed.

Reply:
Makes sense, thanks. I applied your suggestion.